Userland/Games/2048/main.cpp

```diff
@@ -190,7 +186,7 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     })));
 
     TRY(game_menu->try_add_separator());
-    TRY(game_menu->try_add_action(GUI::Action::create("&Settings...", [&](auto&) {
+    TRY(game_menu->try_add_action(GUI::Action::create("&Settings...", Gfx::Bitmap::try_load_from_file("/res/icons/16x16/settings.png").release_value_but_fixme_should_propagate_errors(), [&](auto&) {
```

Comment:
Let’s not add more error propagation debt.
```suggestion
    TRY(game_menu->try_add_action(GUI::Action::create("&Settings...", TRY(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/settings.png")), [&](auto&) {
```

We should apply the same fix above as well,